app/views/user_sessions/_greeting.html.erb

```diff
@@ -6,24 +6,12 @@
         <%= link_to ts("Hi, %{current_user}!", :current_user => current_user.login, key: 'header'), user_path(current_user) %>
         <ul class="menu" role="menu">
           <li><%= span_if_current ts('My Dashboard', key: 'header'), user_path(current_user) %></li>
-          <% if @current_user.subscriptions.count > 0 %>
-            <li><%= span_if_current ts('My Subscriptions', key: 'header'), user_subscriptions_path(current_user) %></li>
-          <% end %>
-          <% if @current_user.visible_work_count > 0 %>
-            <li><%= span_if_current ts("My Works (%{works_number})", key: 'header', :works_number => @current_user.visible_work_count), user_works_path(@current_user) %></li>
-          <% end %>
-          <% if @current_user.bookmarks.count > 0 %>
-            <li><%= span_if_current ts("My Bookmarks (%{bookmark_number})", key: 'header', :bookmark_number => @current_user.bookmarks.count), user_bookmarks_path(current_user) %></li>
-          <% end %>
-          <% if @current_user.owned_collections.count > 0 %>
-            <li><%= span_if_current ts("My Collections (%{coll_number})", key: 'header', :coll_number => @current_user.owned_collections.count), user_collections_path(current_user) %></li>
-          <% end %>
-          <% if @current_user.challenge_signups.count > 0 %>
-            <li><%= span_if_current ts("My Sign-ups (%{signup_number})", key: 'header', :signup_number => @current_user.challenge_signups.count), user_signups_path(current_user) %></li>
-          <% end %>
-          <% if (@current_user.offer_assignments.undefaulted.count || @current_user.pinch_hit_assignments.undefaulted.count) > 0 %>
-            <li><%= span_if_current ts("My Assignments (%{assignment_number})", key: 'header', :assignment_number => (@current_user.offer_assignments.undefaulted.count + @current_user.pinch_hit_assignments.undefaulted.count)), user_assignments_path(current_user) %></li>
-          <% end %>
+          <li><%= span_if_current ts('My Subscriptions', key: 'header'), user_subscriptions_path(current_user) %></li>
```

Comment:
If we are doing that then the counts have already been done and therefore on the ones that have conditionals we could put them back.

I would argue that we should have all the menu items there as it makes it obvious that the feature is available.